report/src/main/webapp/report.html

```diff
@@ -28,7 +28,7 @@
   <script defer src="assets/icons/fontawesome-all.js"></script>
 </head>
 
-<body aet-hide-popovers aet-keyboard-shortcuts aet-sidepanel ng-class="{ 'menu-expanded': sidebarExpanded }">
+<body aet-hide-popovers aet-keyboard-shortcuts aet-sidepanel aet-winter-edition ng-class="{ 'menu-expanded': sidebarExpanded }">
```

Comment:
is this CSS class needed? why? Maybe it should be only added in December ?

Reply:
discussed